src/lib/auth/client.ts

```
@@ -0,0 +1,5 @@
+import { NEXT_BASE_URL } from '@/config/env';
+
+export function getRedirectUri(path: string = '/auth/callback') {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)